### PR TITLE
feat(web): display age in months when age is 0

### PR DESCRIPTION
**Pull request by jpsak09**
I have quite a few baby pictures in my library and being able to see age in months is valuable to me rather than seeing the value 0.
Screenshot 2023-12-24 at 10 35 02 AM

### CODE REVIEW

File: web/src/lib/components/asset-viewer/detail-panel.svelte [hunk 1]
```diff
@@ -277,7 +280,11 @@
                         { locale: $locale },
                       )}
                     >
-                      Age {age}
+                      {#if age == 0 && ageInMonths >= 0 && ageInMonths <= 11}
+                        Age {ageInMonths}mo
```
Comment (alextran1502) [inline web/src/lib/components/asset-viewer/detail-panel.svelte:284]: maybe `{ageInMonth} months old`

File: web/src/lib/components/asset-viewer/detail-panel.svelte [hunk 2]
```diff
@@ -277,7 +280,11 @@
                         { locale: $locale },
                       )}
                     >
-                      Age {age}
+                      {#if age == 0 && ageInMonths >= 0 && ageInMonths <= 11}
```
Comment (danieldietzler) [inline web/src/lib/components/asset-viewer/detail-panel.svelte:283]: You've got some redundant conditions there: if age is 0 ageInMonths will always be <= 11 (per definition). Also, I think it's save to assume that ageInMonths won't be negative (given that we didn't check this atm either).

Comment (jpsak09) [reply web/src/lib/components/asset-viewer/detail-panel.svelte]: Fair points, I simplified the conditions.

**Review by danieldietzler — APPROVED** [review]
LGTM

**Pull request merged**